closefit.R

```diff
@@ -0,0 +1,47 @@
+closefit <- function(m0, m1, df.fix=0) {

```

Comment:
k tomuhle asi mnoho nemam, akorat bych pridal kontrolu, ze `class(m0) / class(m1) == 'merMod`

Reply:
Co znamená to `/` v tom příkazu? Tomu nerozumím...

Reply:
sorry, to neni prikaz, ale mel to byt text, to lomitko je proste "nebo" :)